### DISCUSSION TITLE
Collaborative editing?

### DISCUSSION THREAD

**Discussion by Blakeinstein** [coder/code-server#3853] [Q&A] [answered]
Is it possible (even via extensions) to have collaborative editing possible on the vs code sessions? Like I can pass on the link to my code-server and we can collaborate?

**Comment by jsjoeio** [comment, marked as answer]
Unfortunately, not yet. You can read [this issue](https://github.com/cdr/code-server/issues/33) to see the history (there is one or two OSS solutions which semi-work). We may consider adding something in the future.

**Reply by Blakeinstein** [reply]
That sounds good!

**Comment by lg-nineoneninethreenine** [comment]
> Is it possible (even via extensions) to have collaborative editing possible on the vs code sessions? Like I can pass on the link to my code-server and we can collaborate?

Maybe, since Microsoft made an official extension named Live Share, it might be possible.

**Reply by jsjoeio** [reply]
It's closed source though so we'd have to reimplement it ourselves.
